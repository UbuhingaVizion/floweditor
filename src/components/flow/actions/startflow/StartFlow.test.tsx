import StartFlowComp from 'components/flow/actions/startflow/StartFlow';
import * as React from 'react';
import { render } from 'test/utils';
import { createMsgStartFlowAction } from 'testUtils/assetCreators';

const startFlowAction = createMsgStartFlowAction();

describe(StartFlowComp.name, () => {
  describe('render', () => {
    it('should render flow name', () => {
      const { baseElement, getByText } = render(<StartFlowComp {...startFlowAction} />);
      getByText(startFlowAction.flow.name);
      expect(baseElement).toMatchSnapshot();
    });
  });
});
