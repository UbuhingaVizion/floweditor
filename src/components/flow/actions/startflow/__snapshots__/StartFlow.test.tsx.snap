// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StartFlowComp render should render flow name 1`] = `
<body>
  <div>
    <div
      class="node_asset"
    >
      <div
        style="display: inline-flex;"
      >
        <temba-icon
          classname="node_group"
          name="flow"
          style="margin-right: 4px;"
        />
        <a
          href="/flow/editor/763e4844-3e1b-407a-a1b5-5fdfcd308b41"
          rel="noopener noreferrer"
          target="_blank"
        >
          Colors
        </a>
      </div>
    </div>
  </div>
</body>
`;
